cocotb/drivers/__init__.py

```diff
@@ -311,6 +312,8 @@ def __init__(
     ) -> None:
         BusDriver.__init__(self, entity, name, clock, **kwargs)
         self.set_valid_generator(valid_generator=valid_generator)
+        self.on = None
+        self.off = None
```

Comment:
```suggestion
        self.on = None
        self.off = None
        self.set_valid_generator(valid_generator=valid_generator)
```
since `self.set_valid_generator` overwrites `on`

Reply:
Will do this, with a comment.